### ISSUE TITLE
[Suggestion, Compatibility] Add villager functionality to modded villagers

### ISSUE THREAD

**Issue by Atlantispy** [mactso/Regrowth#37]
Examples of modded villagers could be guard villagers, minecolonies villagers or Minecraft Comes Alive villagers.

That could probably even extend to Orcz or Kobolds

Links to potential mods to add functionality to:
https://www.curseforge.com/minecraft/mc-mods/guard-villagers
https://www.curseforge.com/minecraft/mc-mods/minecolonies
https://www.curseforge.com/minecraft/mc-mods/minecraft-comes-alive-reborn

https://www.curseforge.com/minecraft/mc-mods/orcz
https://www.curseforge.com/minecraft/mc-mods/kobolds

**Comment by mactso** [comment]
This is pretty hard, likely unstable (because the other mods change) but I'll keep it open until i can investigate it.

**Comment by mactso** [comment]
Other mods are too unstable to make this possible.   I can't do this one

**Comment by Atlantispy** [comment]
How is the regrowth functionality being applied, is it specific to the villager mob?

Just thinking potentially you could have it be applying to a tag, and you give that tag to villager mobs but also could be added to other mobs if you see what I mean.

**Comment by D3SL** [comment]
What about just providing some kind of static access to your "XYZ happens" code that any other mod could have their modded villager entities call at appropriate times? If I understand correctly your mod works by just making things happen in the world next to the villager, rather than causing the villager to tangibly do a thing, right?

**Comment by mactso** [comment]
Yes, it works by causing things to happen near the mob or villager.
I am intending to open up villager actions that do not depend on information only villagers (such as where the village center is) know to all mobs in a future release (so any mob could cut grass or trim trees)
